setup.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read().replace('.. :changelog:', '')
+
+requirements = [
+    'wheel',
+    'numpy',
+    'scipy',
+    'matplotlib'
+]
+
+test_requirements = [
+    'pytest'
+]
+
+setup(
+    name='gitgoing',
+    version='0.1.0',
+    description="Get going with open-source contributions",
+    long_description=readme + '\n\n' + history,
+    author="Olga Botvinnik",
+    author_email='olga.botvinnik@gmail.com',
+    url='https://github.com/codeneuro/gitgoing',
+    packages=[
+        'gitgoing',
+    ],
+    package_dir={'gitgoing':
+                 'gitgoing'},
+    include_package_data=True,
+    install_requires=requirements,
+    license="BSD",
+    zip_safe=False,
+    keywords='gitgoing',
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Natural Language :: English',
+        "Programming Language :: Python :: 2",
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
```

Comment:
non `2.7` python versions show up in a few places in this PR. Is that intentional? Are we intending this code to support lots of versions? Is that something we want to do? is it even possible to have python code that is cross compatible across the `2->3` boundary?